services/121-service/test/registrations/bulk-update-registration.test.ts

```diff
@@ -96,16 +111,25 @@ describe('Update attribute of multiple PAs via Bulk update', () => {
     const pa2AfterPatch = searchByReferenceIdAfterPatchPa2.body.data[0];
 
     // Assert
-    assertRegistrationBulkUpdate(
-      registrationDataThatWillChangePa1,
-      pa1AfterPatch,
+    // Explicit assertions for pa1 using patch data
+    expect(pa1AfterPatch).toMatchObject(registrationDataThatWillChangePa1);
+
+    // Explicit assertions for pa2 using patch data
+    expect(pa2AfterPatch).toMatchObject(registrationDataThatWillChangePa2);
+
+    // Ensure unchanged fields remain the same
+    const dataThatStaysTheSamePa1 = filterUnchangedProperties(
       pa1BeforePatch,
+      registrationDataThatWillChangePa1,
     );
-    assertRegistrationBulkUpdate(
```

Comment:
I tried to make the assertions in these tests a bit clearer by removing assertRegistrationBulkUpdate and using toMatchObject instead. The advantages of this approach are:
- You can more easily see the line where an assertion went wrong on a failed test. As it does not point to the assertion helper.
- You can see the key for which the assertion went wrong. In the attached screenshot it's hard to tell if the assertion went wrong for 'phonenumber', 'name' or whatever other property.

![image](https://github.com/user-attachments/assets/62bc28fd-d412-4b6c-98ea-3616298369f5)